viewing_party/main.py

```diff
@@ -0,0 +1,161 @@
+# Project 1 - Viewing party
+# Katrina Kimzey (she|they)
+# Ada Cohort 15 - paper class
+# Started: March 17, 2021
+# Last Updated: March 26, 2021
+
+
+# WAVE ONE 
+# =====================================================================
+
+def create_movie(title, genre, rating):
+    # builds a dictionary with all info
+    # If no movie title/genre/rating given, return None
+    if title and genre and rating:
+        return {
+            "title" : title,
+            "genre" : genre,
+            "rating" : rating
+        }
+    else:
+        return None
+
+def add_to_watched(user_watched_list, movie):
+    # update watched list to include given movie
+    # movie is a dictionary
+    # user_watched_list is a dict
+    user_watched_list["watched"].append(movie)
+    return user_watched_list
+
+def add_to_watchlist(user_to_watchlist, movie):
+    # update list (inside dict) to watch with given movie
+    user_to_watchlist["watchlist"].append(movie)
+    return user_to_watchlist
+
+def watch_movie(user, movie_title):
+    # move from to watch list to watched list
+    # user is a dictionary
+    # movie is a str
+    # DO NOTHING if movie is not in to watch list
+    for film in user["watchlist"]:
+        if film["title"] == movie_title:
+            add_to_watched(user, film)
+            user["watchlist"].remove(film)
+    
+    return user
+
+
+# WAVE TWO
+# =====================================================================
+
+def get_watched_avg_rating(user_data):
+    if not len(user_data["watched"]):
+        return 0
+
+    ratings_sum = 0.0
+
+    for each in user_data["watched"]:
+        print(each["rating"])
+        ratings_sum += each["rating"]
+
+    return ratings_sum/len(user_data["watched"])
+
+def get_most_watched_genre(user_data):
+    # frequency dictionary of genres
+    g_frequency = {}
+
+    # counts each time the genre shows up
+    for each in user_data["watched"]:
+        if each["genre"] in g_frequency:
+            g_frequency[each["genre"]] += 1
+        else:
+            g_frequency[each["genre"]] = 1
```

Comment:
Nice frequency map, I would probably not use `each` ad a loop variable name, maybe `movie` instead.